GetStream/stream-chat-swift
# [CIS-426][CIS-615] Add StreamChat and StreamChatUI to SPM

# What this PR do:
- Support resources for SPM, because Swift 5.3 allows us to do so.
# How to test it
- Run the Demo project (`Integration/SPM_Tests/SwiftPackageManager.xcodeproj`) try to load some resources via UIConfig (for `StreamChatUI` and check if the app doesn't crash after importing  `StreamChat` because it could happen the database model couldn't be find..)

Note: Currently checking out if I can solve the warnings on source files, probably shouldn't be so difficult.
